QualityControlChecker/QualityControlCheckerUKCore.py

```diff
@@ -12,7 +12,12 @@
 import os
 import sys
 
-error=False #used to fail action if any errors found
+'''Prints the repo name'''
+parent = os.path.dirname(path)
+print("Parent directory", os.path.basename(parent))
```

Comment:
Does this need a space so its not Parent directoryUKCORE and instead Parent directory UKCORE?

Reply:
no the , puts a space in. If I had used "Parent directory"+os.path.basename(parent)" then I would have.